src/components/button.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class Button extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    const { content, classes, handleClick } = this.props;
+    return (
+      <button type="button" className={`${classes}`} onClick={handleClick}>{content}</button>
+    );
+  }
+}
+
+Button.propTypes = {
+  content: PropTypes.string.isRequired,
+  classes: PropTypes.string.isRequired,
+  handleClick: PropTypes.func.isRequired,
+};
+
```

Comment:
- [ ] Its  a requirement that at some point in this milestone, you use lifecycle methods other than the `render` method. Kindly implement some lifecycle methods. [Follow this link](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/lessons/class_components_lifecycle.md)

Reply:
Sir i'm not sure I understand what you mean here, because this is a statement from the lifecycle readme

render() is the only mandatory method; all others are optional.